ibm/resource_ibm_schematics_action.go

```diff
@@ -0,0 +1,1665 @@
+										Description: "Is the variable secure or sensitive ?.",
+									},
+									"immutable": &schema.Schema{
+										Type:        schema.TypeBool,
+										Optional:    true,
+										Description: "Is the variable readonly ?.",
+									},
+									"hidden": &schema.Schema{
+										Type:        schema.TypeBool,
+										Optional:    true,
+										Description: "If true, the variable will not be displayed on UI or CLI.",
+									},
+									"options": &schema.Schema{
+										Type:        schema.TypeList,
+										Optional:    true,
+										Description: "List of possible values for this variable.  If type is integer or date, then the array of string will be  converted to array of integers or date during runtime.",
+										Elem:        &schema.Schema{Type: schema.TypeString},
+									},
+									"min_value": &schema.Schema{
+										Type:        schema.TypeInt,
+										Optional:    true,
+										Description: "Minimum value of the variable. Applicable for integer type.",
+									},
+									"max_value": &schema.Schema{
+										Type:        schema.TypeInt,
+										Optional:    true,
+										Description: "Maximum value of the variable. Applicable for integer type.",
+									},
+									"min_length": &schema.Schema{
+										Type:        schema.TypeInt,
+										Optional:    true,
+										Description: "Minimum length of the variable value. Applicable for string type.",
+									},
+									"max_length": &schema.Schema{
+										Type:        schema.TypeInt,
+										Optional:    true,
+										Description: "Maximum length of the variable value. Applicable for string type.",
+									},
+									"matches": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "Regex for the variable value.",
+									},
+									"position": &schema.Schema{
+										Type:        schema.TypeInt,
+										Optional:    true,
+										Description: "Relative position of this variable in a list.",
+									},
+									"group_by": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "Display name of the group this variable belongs to.",
+									},
+									"source": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "Source of this meta-data.",
+									},
+								},
+							},
+						},
+						"link": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Computed:    true,
+							Description: "Reference link to the variable value By default the expression will point to self.value.",
+						},
+					},
+				},
+			},
+			"trigger_record_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "ID to the trigger.",
+			},
+			"state": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
```

Comment:
is the state can be controlled by end user?

Reply:
Updated the attribute. This is due to the schema and the generated code. Have requested an update to the schema